ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -508,6 +520,181 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P12S Engravement of Souls Tracker',
+      type: 'Ability',
+      netRegex: { id: '8305', source: 'Athena', capture: false },
+      run: (data) => ++data.engravementCounter,
+    },
+    // In Engravement 1 (Paradeigma 2), 2 players receive lightTower and 2 players receive darkTower,
+    // 2 players need to guide the light beam and 2 players need to guide the dark beam.
+    // The operator of the beam extends the beam directly from the outside. The beam is attenuated until the jagged line disappears.
+    // The people in the tower find the people who have the opposite attribute to the debuff and put them in four places.
+    // At NE NW SE SW as a # shape. The position of outside Anthropos is fixed by two situation.
+    // {[97, 75], [125, 97], [103, 125], [75, 103]} and {[103, 75], [125, 103], [97, 125], [75, 97]}. The Anthropos will cast
+    // 'Searing Radiance' for light beam and 'Shadowsear' for dark beam. We use those as a trigger for Tower players place
+    // the Tower.
+    // When debuffs expire and towers drop, their debuff changes to lightTilt or darkTilt (same as tower color).
+    // At the same time the towers drop, the 4 tethered players receive lightTilt or darkTilt depending on their tether color.
+    //
+    {
+      id: 'P12S Engravement 1 Drop Tower Pos Tracker',
+      type: 'StartsUsing',
+      netRegex: { id: ['82F[12]'], source: 'Anthropos' },
+      condition: (data) => data.engravementCounter === 1,
+      promise: async (data, matches) => {
+        data.combatantData = [];
+        const ids = parseInt(matches.sourceId, 16);
+        if (ids === undefined)
+          return;
+        data.combatantData = (await callOverlayHandler({
+          call: 'getCombatants',
+          ids: [ids],
+        })).combatants;
+      },
+      run: (data, matches) => {
+        const x = data.combatantData[0]?.PosX;
+        if (x === undefined)
+          return;
+        const y = data.combatantData[0]?.PosY;
+        if (y === undefined)
+          return;
+
+        if (x === 75 && y === 97) {
+          if (matches.id === '82F1')
+            data.engravement1BeamsPosMap.set('NE', 'dark');
+          else
+            data.engravement1BeamsPosMap.set('NE', 'light');
+        } else if (x === 97 && y === 75) {
+          if (matches.id === '82F1')
+            data.engravement1BeamsPosMap.set('SW', 'dark');
+          else
+            data.engravement1BeamsPosMap.set('SW', 'light');
+        } else if (x === 103 && y === 75) {
```

Comment:
This is probably my biggest request here, but I think there are different ways to solve this mechanic.  The PF strat (which you have here) is to for the tether people to take the tower that's in their quadrant, i.e. the person tethered to this NNE (103, 75) add goes CCW to their tower SE, but if the add is NNW (97, 75) they go CW to their tower SW.  I've seen a different strategy where tethers always rotate clockwise, e.g. people tethered to either NNE or NNW always take the SW tower.

This will likely make this a little more complicated, but can you add some triggerset config ui for this, and add three options?
* no strategy (default, just calls out "drop light tower" with no position)
* quadrant strategy (this PR, maybe PF has a better name for this)
* clockwise strategy (tethers always rotate clockwise to take towers)

...and then `P12S Engravement 1 Drop Tower Pos Tracker` will have to check the option and then set slightly different values in pos map.

If you haven't seen the triggerset config ui, it looks like:
* https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts#L40
* https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts#L173-L194
* https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts#L1909

(let me know if I can explain anything else)

Reply:
The strategy of this implement is based on game8.jp, Which is wildly used in Japan players. I think triggerset config ui is great for this condition. I'm gonna go check on the current solution for this.

Reply:
Both tilt and Vertical methods are implemented with reference to the TOP trigger, while providing the option to display only the type of tower, not the position. 

https://github.com/quisquous/cactbot/pull/5597/commits/cb16f2617173a329a7988307b02cbb346a5f09f6